### PR TITLE
[Fuzzing] remove old go-fuzz and use Native Go fuzz

### PR DESCRIPTION
**Pull request by 0x34d**
[go-fuzz](https://github.com/dvyukov/go-fuzz) is deprecated in favor of https://go.dev/security/fuzz

* using Native [Go fuzz](https://go.dev/security/fuzz)
* Add fuzzing seed.
* Support of oss-fuzz infrastructure.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+fuzz:
```
Comment (pmazzini) [inline Makefile:1]: Do we really need to commit this Makefile?

Comment (0x34d) [reply Makefile:1]: Not actually

Comment (pmazzini) [reply Makefile]: Should we run them as part of the CI?

Comment (0x34d) [reply Makefile]: >Should we run them as part of the CI? 

My proposal is a little bit longer but it's better for the DHCP project.
See, the DHCP project is used by a lot of [projects](https://github.com/insomniacslk/dhcp#public-projects-that-use-it), so I can integrate this into [oss-fuzz](https://google.github.io/oss-fuzz/) and later [CIFuzz](https://google.github.io/oss-fuzz/getting-started/continuous-integration/) can be added.

Steps are:
* Merge this PR.
* Integrate oss-fuzz.
* Implement CIFuzz for DHCP PR.

File: README.md [hunk 2]
```diff
@@ -42,6 +42,15 @@ packages.
 
 See more example code at https://github.com/insomniacslk/exdhcp
 
+# Fuzzing
```
Comment (pmazzini) [inline README.md:45]: Do we really need this? This is not different to what the `go-fuzz` docs mention.

Comment (0x34d) [reply README.md]: ReMoved

**Comment by 0x34d** [comment]
Request: @pmazzini, may I have your email ID for OSS-Fuzz `primary_contact`?

**Comment by 0x34d** [comment]
> Request: @pmazzini, may I have your email ID for OSS-Fuzz `primary_contact`?

USING : https://insomniac.slackware.it/pages/email

**Pull request merged**